javascript/ql/test/library-tests/frameworks/SQL/better-sqlite3.js

```diff
@@ -0,0 +1,9 @@
+const db = require('better-sqlite3')('BetterSqlite.db', { verbose: console.log });
+
+const sql = 'SELECT name, id FROM table1'
+const stmt = db.prepare(sql);
+let exec = db.exec(sql);
```

Comment:
Why two test files for `better-sqlite3`?  
They're basically identical except for the file-extensions.  
It could confuse people to think the file-extensions matter, so I think it's better if you combine them into one file.